src/MuTalk-Model/MTAnalysis.class.st

```diff
@@ -151,6 +159,9 @@ MTAnalysis >> initialTestRun [
 
 	| results |
 	results := testCases collect: [ :aTestCase | aTestCase run ].
+	
+	"The test filter is initialized here because the default one needs the run time of tests, so the initial test run must be executed first"
+	self initializeTestFilter.
 	testFilter validateFailuresIn: results.
```

Comment:
```suggestion
	"The test filter is initialized lazyly because the default one needs the run time of tests, so the initial test run must be executed first"
	self testFilter validateFailuresIn: results.
```